### DISCUSSION TITLE
HMIPServer crash und permanenter Neustart

### DISCUSSION THREAD

**Discussion by franky121** [OpenCCU/OpenCCU#2777] [🇩🇪 Fragen&Antworten]
Hi,
habe seit einigen Wochen Probleme mit der RaspberryMatic bzw. dem HMIPServer. Die Box läuft für eine Zeit ohne Probleme und dann fängt der HMIPServer an ständig neu zu starten. Die Hardware incl. Speicherkarte und Netzteil ist komplett getauscht. HomeAssistant hatte ich bereits testweise deaktiviert da ich den im Verdacht hatte. 
Die Neustarts erfolgen dann immer öfter und am Ende wird die Kiste so heiß das ich damit Grillen kann (was theoretisch eine gute Idee wäre :) ) 

Aus dem Log werde ich nicht schlau bzw. würde mich über Schwarmwissen freuen wo ich noch gucken kann: 

Hier ein Auszug:

Jun 16 04:34:56 homematicccu3 user.err monit[1921]: 'temperature' status failed (20) -- 81.668
Jun 16 04:35:13 homematicccu3 user.info monit[1921]: 'HMIPServer' process is running after previous exec error (slow starting or manually recovered?)
Jun 16 04:35:13 homematicccu3 user.err monit[1921]: 'HMIPServer' failed protocol test [DEFAULT] at [localhost]:39292 [TCP/IP] -- Connection refused
Jun 16 04:35:13 homematicccu3 user.info monit[1921]: 'HMIPServer' trying to restart
Jun 16 04:35:13 homematicccu3 user.info monit[1921]: 'HMIPServer' restart: '/etc/init.d/S62HMServer restart'
Jun 16 04:35:43 homematicccu3 user.err monit[1921]: 'HMIPServer' failed to restart (exit status -1) -- '/etc/init.d/S62HMServer restart': Program timed out -- Stopping HMIPServer: OK Starting HMIPServer: ...............
Jun 16 04:35:44 homematicccu3 user.err monit[1921]: 'temperature' status failed (20) -- 81.668
Jun 16 04:35:53 homematicccu3 kern.info kernel: [557014.775329] eq3loop: eq3loop_write_master() return error: -14
Jun 16 04:35:53 homematicccu3 kern.err kernel: [557147.514257] eq3loop: eq3loop_write_master() mmd_hmip: not enough space in buffers. free space = 4, required space = 48
Jun 16 04:36:01 homematicccu3 user.err monit[1921]: 'HMIPServer' service restarted 1 times within 1 cycles(s) - exec
Jun 16 04:36:01 homematicccu3 user.info monit[1921]: 'HMIPServer' exec: '/bin/triggerAlarm.tcl HMIPServer restarted WatchDog: hmipserver-restart true'
Jun 16 04:36:01 homematicccu3 user.info monit[1921]: 'HMIPServer' process is running after previous exec error (slow starting or manually recovered?)
Jun 16 04:36:01 homematicccu3 user.info monit[1921]: 'HMIPServer' process is running after previous restart timeout (manually recovered?)
Jun 16 04:36:01 homematicccu3 user.err monit[1921]: 'HMIPServer' failed protocol test [DEFAULT] at [localhost]:39292 [TCP/IP] -- Connection refused
Jun 16 04:36:01 homematicccu3 user.info monit[1921]: 'HMIPServer' trying to restart
Jun 16 04:36:01 homematicccu3 user.info monit[1921]: 'HMIPServer' restart: '/etc/init.d/S62HMServer restart'

**Comment by gitpit0815** [comment]
Ich habe das gleiche Problem, gibts dazu schon was neues?
